Source/Resilience/ICanDefineDefaultPolicy.cs

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+namespace Dolittle.Resilience
+{
+    /// <summary>
+    /// Defines a system that is capable of defining the default policy for resilience
+    /// </summary>
+    public interface ICanDefineDefaultPolicy
```

Comment:
I would suggest changing the name to something along the lines of "IDefineTheDefaultPolicy", since it does definitely define a policy and there can only be one.

Reply:
Very good point.  Will do.